src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1401,12 +1428,13 @@ public Conversion ClassifyImplicitExtensionMethodThisArgConversion(BoundExpressi
 
             if ((object)sourceType != null)
             {
+                Debug.Assert(!IncludeNullability); // PROTOTYPE(NullableReferenceTypes): Add test.
                 var tupleConversion = ClassifyTupleConversion(
                     sourceType,
                     destination,
                     ref useSiteDiagnostics,
                     ConversionKind.ImplicitTuple,
-                    (ConversionsBase conversions, TypeSymbol s, TypeSymbol d, ref HashSet<DiagnosticInfo> u, bool a) => conversions.ClassifyImplicitExtensionMethodThisArgConversion(null, s, d, ref u),
+                    (ConversionsBase conversions, TypeSymbolWithAnnotations s, TypeSymbolWithAnnotations d, ref HashSet<DiagnosticInfo> u, bool a) => conversions.ClassifyImplicitExtensionMethodThisArgConversion(null, s.TypeSymbol, d.TypeSymbol, ref u), 
```

Comment:
>=> [](start = 147, length = 2)

Nit: Consider breaking this line (and possibly the one on line 1421 too)

Reply:
Done.